popup-app/index.js

```diff
@@ -22,7 +22,7 @@ findAppOrigin()
     let element
     if (!appOrigin) {
       element = <NoParent appName={appName} />
-    } else if (window.location.hash) {
+    } else if (window.location.search) {
```

Comment:
It's not breaking if you upgrade your popup application as well. Do you think that's too much to ask?

Reply:
That seems fine, we probably just want to mention.
However, I'm concerned that applications that _don't_ use the popup will break.

Then again, this is a semver.major? Or not?

Reply:
It would be great if we could make this minor. I considered window.location.search to be an internal thing since only the popup cares about it, and it's automatically built.

Reply:
Yes, the popup I don't find too concerning. It's rather the code that does _not_ use popup. But I take it from your comments that this one is not affected.